### PR TITLE
ENH: Issue #2679. DataFrame.to_html() to create hyperlinks for valid URL...

### LINKED ISSUE #2679: Improvement: DataFrame.to_html() to create hyperlinks
When the data value is a string that represents an URL, have DataFrame.to_html() create hyperlinks out of them in the resulting html file.

### PR DESCRIPTION
**Pull request by tdas14**
Closes #2679

### CODE REVIEW

**Comment by shoyer** [comment]
how does this handle the automatic HTML pretty-printing of dataframes in the ipython notebook? is the presentation of URLs affected?

**Comment by tdas14** [comment]
Aa far as I know it shouldn't. All it does is check for a valid url string and then put the value in an 
<a> tag.

**Comment by shoyer** [comment]
would you mind doing a quick sanity check of the notebook output under your patch? e.g., this is what I currently see:
screen shot 2015-04-13 at 4 20 57 pm

**Comment by shoyer** [comment]
For what it's worth, I don't think it would be a bad thing for the notebook output to include URLs as links, but it would definitely be annoying if they are pretty-printed as a HTML code inside the table :).

**Comment by kay1793** [comment]
this should to be optional at the very least if not off by default for backward compatibility.
`to_html` is not used solely for ipnb display.

**Comment by jreback** [comment]
@tdas14 this needs to be an option when printing (e.g. passed to ``.to_html``).

File: pandas/tests/test_format.py [hunk 1]
```diff
@@ -716,6 +716,35 @@ def test_to_html_multiindex_sparsify_false_multi_sparse(self):
 </table>"""
             self.assertEqual(result, expected)
 
+    
+    def test_to_html_with_hyperlinks(self):
+        df = DataFrame([[0,'http://pandas.pydata.org/', 'pydata.org']],columns=['foo', 'bar', None], index=lrange(1))
```
Comment (jreback) [inline pandas/tests/test_format.py]: add the issue number here

**Comment by jreback** [comment]
@tdas14 this needs to be an option directly to ``.to_html``, further, you could add a global option in ``display.html.render_links``, as a boolean (default should be False for back-compat)

**Comment by jreback** [comment]
closing, but if you would like to continue working on this pls reopen

**Pull request closed without merging**